Source/Extensions/UIViewControllerExtensions.swift

```diff
@@ -53,3 +53,20 @@ extension UIViewController {
         notificationCenter.removeObserver(self, name: UIResponder.keyboardWillHideNotification, object: nil)
     }
 }
+
+public extension UIViewController {
+    func customizeNavigationBarAppearance() {
+        if #available(iOS 15.0, *) {
+            let appearance = UINavigationBarAppearance()
```

Comment:
I don't think this extension should be public. The SDK should not offer the consumer UI formatting behaviours. Furthermore:
- are you sure the problem this code solves is not an issue on other OS versions, or you just haven't checked yet
- `UINavigationBar.appearance()` is a singleton type of behaviour. This code is hijacking the end users UI, I don't think any SDK consumer would appreciate this hidden behaviour

Reply:
I am still not convinced this should be public.

I am not sure the problem its trying to solve, nor that it works pre iOS-15? Will try to find time to check, if I can figure out how 🙈